### DISCUSSION TITLE
Machine Learning Podcasts!

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#22] [General help]
:question: **What are your favorite Machine Learning podcasts?**

In case you didn’t know, we love podcasts-- you can check our podcast, Voice of the Future: Our Favorite Nerds [here](https://deepgram.com/voiceofthefuturepodcast/) :wink: 

Here are four others we’ve enjoyed: 

- [‎Machine Learning on Apple Podcasts](https://podcasts.apple.com/us/podcast/machine-learning/id384233048) 
- [‎Machine Learning – Software Engineering Daily on Apple Podcasts](https://podcasts.apple.com/us/podcast/machine-learning-software-engineering-daily/id1230807136) 
- [‎Machine Learning Street Talk (MLST) on Apple Podcasts](https://podcasts.apple.com/us/podcast/machine-learning-street-talk-mlst/id1510472996) 
- [‎Adventures in Machine Learning on Apple Podcasts](https://podcasts.apple.com/us/podcast/adventures-in-machine-learning/id1531074814)

**Comment by Gift-Ojeabulu** [comment]
## My Top Data/Machine learning Podcast

- [DataTalksClubs](https://datatalks.club/)
- [TheDatataScientistShow](https://www.youtube.com/c/TheDataScientistShow/videos)
- [SuperDataScience](https://www.superdatascience.com/podcast)
- [Datapodchat](https://twitter.com/datapodchat)

**Reply by BekahHW** [reply]
Can't wait to check these out! Thanks for sharing, @Gift-Ojeabulu.
